src/web/assets/cp/src/js/TableElementIndexView.js

```diff
@@ -19,9 +21,20 @@ Craft.TableElementIndexView = Craft.BaseElementIndexView.extend({
     },
 
     afterInit: function() {
+        // Set table caption
+        this.$tableCaption = this.$table.find('caption');
+
+        this.$statusMessage = this.$table.parent().find('[data-status-message]');
+
         // Set the sort header
         this.initTableHeaders();
 
+        // Add callback for after elements are updated
+        this.elementIndex.settings.onUpdateElements = () => {
+            this.$table.find('th [aria-pressed="true"]').focus();
```

Comment:
`onUpdateElements` seems to run the first time an element select modal is opened as well. Hoping to focus on the pressed button only after a user explicitly sorts by pressing a button - any suggestions? Tried adding to the `handleSelectedSortHeaderClick` and `handleUnselectedSortHeaderClick` functions, but couldn't time them to run after the element update.

Reply:
Fixed by just registering an `updateElements` event:

https://github.com/craftcms/cms/blob/0d32e98121cb819d5b1eb1d52955d66c2cf55bb5/src/web/assets/cp/src/js/TableElementIndexView.js#L33

Reply:
Fixed this from the base element index class, which now captures the active element before it begins swapping out the element index view with the new one:

https://github.com/craftcms/cms/blob/0d32e98121cb819d5b1eb1d52955d66c2cf55bb5/src/web/assets/cp/src/js/BaseElementIndex.js#L1762-L1763

Then checks if that element is no longer part of the DOM, and if so, brings focus back to a new element with the same ID (if one exists):

https://github.com/craftcms/cms/blob/0d32e98121cb819d5b1eb1d52955d66c2cf55bb5/src/web/assets/cp/src/js/BaseElementIndex.js#L1920-L1922